automatminer/featurization/core.py

```diff
@@ -220,7 +220,8 @@ def __init__(self, preset=None, featurizers=None, exclude=None,
                  use_metaselector=False, functionalize=False, max_na_frac=0.05,
                  ignore_cols=None, ignore_errors=True, drop_inputs=True,
                  guess_oxistates=True, multiindex=False, n_jobs=None,
-                 logger=True):
+                 logger=True, composition_col="composition", structure_col="structure",
```

Comment:
Could we change bandstructure and dos to bandstructure_col and dos_col?

Reply:
Sounds good. It's ready to go after this